src/stack_manager.py

```diff
@@ -80,35 +88,28 @@ def parse(self, tag_list: List[List]) -> List:
 
         self.lookaheads = 0
         for tag, *scoped_tags in tag_list:
-            if is_layout_obj(tag):  # create new scope for layout object
-                if type(tag) == LPText:
-                    self.current_scope.children.append(Scope(tag, scoped_tags))
-                elif type(tag) == LPWhitespacePrefix:
-                    # TODO: Figure out dependencies between auto and lookahead and also actually program them in somewhere (here vs backend?)
```

Comment:
Has this TODO been taken care of? If not, it should be left in

Reply:
put back in